### PR TITLE
update multi-device section

### LINKED ISSUE #628: update "multidevice" in help
- adapt to the new, recommented flow
- add a troubleshoting paragraph (apps link to `https://delta.chat/en/help#multiclient` with the menu item or button "Troubleshooting")
- add the current flow as alternative path, at best leaving things mostly as is, to avoid forcing retranslation

### PR DESCRIPTION
**Pull request by r10s**
describe 'Add Second Device' in help, add troubleshooting section and leave the old method as an alternative.

if we get that in soon, we can even add it to the offline help before 1.36 is shipped

closes #628

### CODE REVIEW

File: en/help.md [hunk 1]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+- On the new device, install Delta Chat (<https://get.delta.chat>)
+
+- Start Delta Chat on the new device, select **Add as Second Device** and scan the QR code from the old device
+
+- Transfer should start after a few seconds and during transfer both devices will show the **progress**.
+  Wait until it is finished on both devices.
+
+In contrast to many other messengers, after successful transfer,
+both **devices are completely independent.**
+The one device is not needed for the other to work.
```
Comment (adbenitez) [inline en/help.md]: maybe "the first device", "the one device" sounds weird to me, but maybe just because I am no native speaker

Comment (adbenitez) [reply en/help.md]: or "One device is not needed for another to work" maybe

**Review by adbenitez — APPROVED** [review]

File: en/help.md [hunk 2]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+
+
+### Troubleshooting
```
Comment (missytake) [inline en/help.md:459]:
```suggestion
#### Troubleshooting
```

This should be a sub-headline of "Can I use Delta Chat on multiple devices?"

File: en/help.md [hunk 3]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+
+- Double-check both devices are in the **same Wi-Fi or network**
+
+- Your system may have a "personal firewall",
+  this is known to make problems, esp. on Windows.
```
Comment (missytake) [inline en/help.md]:
```suggestion
  this is known to cause problems, especially on Windows.
```
slightly better wording.

File: en/help.md [hunk 4]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+In contrast to many other messengers, after successful transfer,
+both **devices are completely independent.**
+The one device is not needed for the other to work.
+
+
+### Troubleshooting
+
+- Double-check both devices are in the **same Wi-Fi or network**
+
+- Your system may have a "personal firewall",
```
Comment (missytake) [inline en/help.md]:
```suggestion
- Your system might have a "personal firewall",
```
"may" has the connotation of "is allowed to" - in this context we mean the opposite.

File: en/help.md [hunk 5]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+  this is known to make problems, esp. on Windows.
+  **Disable the personal firewall** for Delta Chat on both ends and try again
+
+- Ensure there is **enough storage** on the destination device
+
+- If transfer started, make sure, the devices **stay active** and do not fall asleep.
+  Do not exit Delta Chat.
+  (we try hard to work in background, but systems tend to kill apps, unfortunately)
```
Comment (missytake) [inline en/help.md]:
```suggestion
  (we try hard to make the app work in background, but [systems tend to kill apps](https://dontkillmyapp.com), unfortunately)
```

File: en/help.md [hunk 6]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+
+- Delta Chat is **already installed** on the destination device?
```
Comment (missytake) [inline en/help.md]:
```suggestion
- Delta Chat is **already logged in** on the destination device?
```
Let's keep it precise. People who installed the app according to our guide, but didn't login yet, could think this point applies to them.

I admit that my suggestion has a slightly weird wording. But technically, it *is* Delta Chat that is logged in to the email account :sweat_smile: maybe someone else can make up something better.

File: en/help.md [hunk 7]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+  You can use multiple accounts per device, just [add another account](#multiple-accounts)
+
+- If you still have problems or if you **cannot scan a QR code**
+  try the **manual transfer** described below
+
+
+### Manual Transfer
```
Comment (missytake) [inline en/help.md:480]:
```suggestion
#### Manual Transfer
```
Should be a sub-headline as well.

**Review by missytake — COMMENTED** [review]
All of this is a great improvement! I've been looking forward to this for years. See suggestions for my nitpicks :D

File: en/help.md [hunk 8]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
-If you want to use the **same account** on different devices, you should export
-a backup from the old device, and import it into the new device:
+Yes. Delta Chat 1.36 comes with a new, experimental function for using the same account on different devices:
+
+- On the old device, go to **Settings → Add Second Device**, unlock the screen if needed
+  and wait a moment until a QR code is shown
+
+- On the new device, install Delta Chat (<https://get.delta.chat>)
+
+- Start Delta Chat on the new device, select **Add as Second Device** and scan the QR code from the old device
```
Comment (rosano) [inline en/help.md]:
```suggestion
- On the first device, go to **Settings → Add Second Device**, unlock the screen if needed
  and wait a moment until a QR code is shown

- On the second device, [install Delta Chat](https://get.delta.chat)

- On the second device, start Delta Chat, select **Add as Second Device**, and scan the QR code from the old device
```

To me old/new implies transferring data for single-device usage.

File: en/help.md (hunk 1 above)
Comment (rosano) [reply en/help.md]:
```suggestion
One device is not needed for the other to work.
```

File: en/help.md [hunk 9]
```diff
@@ -439,8 +439,47 @@ For other programs, you should be able to find a solution online.
...
+both **devices are completely independent.**
+The one device is not needed for the other to work.
+
+
+### Troubleshooting
+
+- Double-check both devices are in the **same Wi-Fi or network**
+
+- Your system might have a "personal firewall",
+  this is known to cause problems, especially on Windows.
```
Comment (rosano) [inline en/help.md]:
```suggestion
- Your system might have a "personal firewall",
  which is known to cause problems (especially on Windows).
```

File: en/help.md (hunk 2 above)
Comment (r10s) [reply en/help.md:459]: in theory, yes, maybe. but one can see this also for "troubleshooting for multidevice" - "enough storage", "already installed?" eg. also fits for "manual transfer".

in practise, we do not have 4th level headlines in the FAQ and it does not look good in the content (small screens). so, i  think, we should be too bureaucratic here, it is just the next section.

**Comment by r10s** [comment]
yeah, thanks all ❤️

**Pull request merged**